internal/commands/push/command.go

```diff
@@ -100,16 +100,17 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	var uploadPathDependencies string
+	var uploadDependenciesPath string
```

Comment:
[question] did this need to change?  no need to revert if not, just going forward its generally advisable to limit the amount of drive-bys in a PR to keep the diff focused and easier to read.

some instances where a drive-by may be worth it: if its fixing an issue or otherwise undoing a pattern we're trying to eliminate from a codebase

Reply:
I just found it to be confusing to be called `PathDependencies` as opposed to `DependenciesPath` and worthy of changing to reflect a causal connection with the `uploadDependencies` variable I declared below. Will take the note about drive by-s in mind in future PRs, thank you! I think it will be good to change this to `appDependenciesPath`, again to be consistent with the other suggested variable change.

Reply:
naming in go can get weird at times (as with all languages).  you'll notice we use `uploadPath` around the codebase, and the intent here was to specify/denote the flavor of `uploadPath` we were dealing with, in this case a dependencies one.

either way, feel free to leave as you have it now!  just wanted to chime in about drive-bys in general here